### PR TITLE
[Feature][Core] shell batch cancel task

### PR DESCRIPTION
**Pull request by zhangshenghang**
### Purpose of this pull request

shell command batch cancel task

- reslove #7512

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-e2e/seatunnel-core-e2e/seatunnel-starter-e2e/src/test/java/org/apache/seatunnel/core/starter/seatunnel/SeaTunnelConnectorBatchCancelTest.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                        .map(
+                                s -> {
+                                    Matcher matcher = pattern.matcher(s);
+                                    return matcher.find() ? matcher.group(1) : null;
+                                })
+                        .filter(jobId -> jobId != null)
+                        .collect(Collectors.toList());
+
+        // Verify that the status is Running
+        for (String jobId : runningJobId) {
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-core-e2e/seatunnel-starter-e2e/src/test/java/org/apache/seatunnel/core/starter/seatunnel/SeaTunnelConnectorBatchCancelTest.java:106]: Please add assert of `runningJobId` count.

File: docs/en/seatunnel-engine/user-command.md [hunk 2]
```diff
@@ -114,9 +114,11 @@ Both failed jobs and jobs paused by seatunnel.sh -s &lt;jobId&gt; can be resumed
 ## Canceling Jobs
 
 ```shell
-sh bin/seatunnel.sh -can &lt;jobId&gt;
+sh bin/seatunnel.sh -can <jobId1> [<jobId2> <jobId3> ...]
```
Comment (hailin0) [inline docs/en/seatunnel-engine/user-command.md:117]: Can you support rest api?

Comment (zhangshenghang) [reply docs/en/seatunnel-engine/user-command.md:117]: > Can you support rest api?

@hailin0  Supported in  #7522

File: seatunnel-e2e/seatunnel-core-e2e/seatunnel-starter-e2e/src/test/java/org/apache/seatunnel/core/starter/seatunnel/SeaTunnelConnectorBatchCancelTest.java (hunk 1 above)
Comment (zhangshenghang) [reply seatunnel-e2e/seatunnel-core-e2e/seatunnel-starter-e2e/src/test/java/org/apache/seatunnel/core/starter/seatunnel/SeaTunnelConnectorBatchCancelTest.java:106]: already added

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @zhangshenghang !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**